src/app/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import i18next from 'i18next';
+import { useForm } from 'react-hook-form';
+
+import styles from './styles.module.scss';
+import WoloxImg from './assets/wolox-logo.png';
+
+interface User {
+  first_name: string;
+  last_name: string;
+  email: string;
+  password: string;
+  password_confirmation?: string;
+  locale?: string;
+}
+
+const SignUp: React.FC = () => {
+  const { register, handleSubmit, watch, errors } = useForm<User>({ mode: 'all' });
+  const onSubmit = handleSubmit(data => {
+    data.locale = i18next.language;
+    console.log(data);
+  });
+
+  return (
+    <div className="row center">
+      <form onSubmit={onSubmit} className={`column ${styles.signupForm}`}>
+        <div className={styles.header} />
+        <div className={styles['wolox-logo-wrapper']}>
+          <img src={WoloxImg} alt={i18next.t('SignUp:logoAlt') as string} className={styles['wolox-logo']} />
+        </div>
+        <label className={styles['custom-labels']}>{i18next.t('SignUp:firstName')}</label>
+
+        <input
+          name="first_name"
+          className={styles['custom-inputs']}
+          ref={register({
+            required: { value: true, message: i18next.t('SignUp:required') }
+          })}
+        />
+        {errors.first_name && <label className={styles['error-labels']}>{errors.first_name.message}</label>}
+        <label className={styles['custom-labels']}>{i18next.t('SignUp:lastName')}</label>
+        <input
+          name="last_name"
+          className={styles['custom-inputs']}
+          ref={register({
+            required: { value: true, message: i18next.t('SignUp:required') }
+          })}
+        />
+        {errors.last_name && <label className={styles['error-labels']}>{errors.last_name.message}</label>}
```

Comment:
this logic will be repeated across al the app, what about making a custom `Input` component, that receives type, ref, classname, name, error, type, etc.